express-hello-world/index.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+
+const app = express();
+
+app.use(function (req, res, next) {
+  // console.log('Time:', Date.now())
+  // console.log(req.method)
+  // console.log(res.method)
+  res.send('test send');
+  next();
```

Comment:
We don't want to mix `res.send` and `next`. We always want to use one or the other. In this case we just want to use `res.send`. We will discuss the differences and why in more detail in the Q and A.